### PR TITLE
URL Shortening

### PR DESCRIPTION
**Pull request by bruno-garcia**
* Migrations wasn't tested during 3.0 migration and seems broken.
* I suggest we just move to 3.1 (wont' build with sdk 3.0.100 anyway)
* Suggest go live building `redirectr` and only after a few iterations (bugfixes and security patches) we publish packages and acquire via nuget.
* npgsql needs some better way to configure the DI container
* We'd add the schema changes to EF migration and set the options:createSchemaFalse=false

### CODE REVIEW

File: src/NuGetTrends.Data/NuGetTrends.Data.csproj [hunk 1]
```diff
@@ -11,6 +11,7 @@
 
   <ItemGroup>
     <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="2.2.4" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="2.2.6" />
```
Comment (bruno-garcia) [inline src/NuGetTrends.Data/NuGetTrends.Data.csproj]: need to fix breaking changes to go to 3.0+ but even on 2.2 it's failing to run.

Comment (joaopgrassi) [reply src/NuGetTrends.Data/NuGetTrends.Data.csproj]: what's the issue? I remember running during the last PR (where we upgraded to 3.0) and it seemed to work. But not 100% sure anymore. Anyway moving to 3.1 is probably the best thing.

Comment (bruno-garcia) [reply src/NuGetTrends.Data/NuGetTrends.Data.csproj]: Last error I had was: `Unable to create an object of type 'NuGetTrendsContext'. For the different patterns supported at design time, see https://go.microsoft.com/fwlink/?linkid=851728`

**Comment by bruno-garcia** [comment]
CLosed in favor of #87

**Pull request closed without merging**